nixpkgs_review/cli/__init__.py

```diff
@@ -134,18 +134,6 @@ def read_github_token() -> str | None:
     token = os.environ.get("GITHUB_OAUTH_TOKEN", os.environ.get("GITHUB_TOKEN"))
     if token:
         return token
-    try:
-        with hub_config_path().open() as f:
-            for line in f:
-                # Allow substring match as hub uses yaml. Example string we match:
-                # " - oauth_token: ghp_abcdefghijklmnopqrstuvwxyzABCDEF1234\n"
-                token_match = re.search(
-                    r"\s*oauth_token:\s+((?:gh[po]_)?[A-Za-z0-9]+)", line
```

Comment:
This removes reading of the token from the `hub` tool. It looks unrelated to `ofborg` removal. Is the removal intentional?

Reply:
Yes we had several instances where this token was out of date. Also hub now often stores git hub tokens in platform specific keychains. Using GH is better because the token than should always work and not leave the user with a 403 error.

Reply:
Aha, thank you! I'll try to migrate from `hub` to `gh`.